Sources/Sentry/include/SentryProfilerState.h

```diff
@@ -6,9 +6,22 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * Parses a symbol that is returned from @c backtrace_symbols()  which encodes information
+ * like the frame index, image name, function name, and offset in a single string.
+ * @discussion For the input:
+ *  @code
+ * 2 UIKitCore 0x00000001850d97ac -[UIFieldEditor _fullContentInsetsFromFonts] + 160
+ * @endcode
+ * This function would return:
+ * @code -[UIFieldEditor _fullContentInsetsFromFonts] @endcode
+ * @note If the format does not match the expected format, this returns the input string.
+ */
+NSString *parseBacktraceSymbolsFunctionName(const char *symbol);
+
 @class SentrySample;
 
-@interface SentryProfilingMutableState : NSObject
+@interface SentryProfilerMutableState : NSObject
```

Comment:
I'd added this to a file called SentryProfilerState but the interface name was SentryProfilingState. Aligning them both to reduce confusion.